### PR TITLE
The spraycan now works on walls again.

### LINKED ISSUE #38763: Graffiti on walls don't show up
If you spray graffiti on a wall it gives you a success message and you lose a charge on the spray can but the graffiti doesn't appear

### PR DESCRIPTION
**Pull request by ShovelTime**
Fixes #38763

:cl:
fix: Spraycans are now able to spray walls again!
/:cl:

Graffities will no longer disappear once sprayed on walls.

### CODE REVIEW

**Comment by Gousaid67** [comment]
Travis will be broke till the BYOND hub works again, will rerun checks then

**Comment by Dorsisdwarf** [comment]
Cheers

File: code/game/objects/effects/decals/crayon.dm [hunk 1]
```diff
@@ -28,3 +28,6 @@
 	if(main)
 		paint_colour = main
 	add_atom_colour(paint_colour, FIXED_COLOUR_PRIORITY)
+
+/obj/effect/decal/cleanable/crayon/NeverShouldHaveComeHere(turf/T)
```
Comment (Cyberboss) [inline code/game/objects/effects/decals/crayon.dm]: nigga wtf is this proc name?

**Review by Cyberboss — APPROVED** [review]

File: code/game/objects/effects/decals/crayon.dm (hunk 1 above)
Comment (Gousaid67) [reply code/game/objects/effects/decals/crayon.dm]: no bloody clue, check /obj/effect/decal, im simply overriding it here.

Comment (Mickyan) [reply code/game/objects/effects/decals/crayon.dm]: wanna hear a funny joke https://github.com/tgstation/tgstation/pull/34121

Comment (subject217) [reply code/game/objects/effects/decals/crayon.dm]: wew lad

Comment (Cyberboss) [reply code/game/objects/effects/decals/crayon.dm]: https://www.youtube.com/watch?v=Lr7CKWxqhtw

**Pull request merged**